packages/a/ada/xmake.lua

```diff
@@ -0,0 +1,34 @@
+package("ada")
+
+    set_homepage("https://ada-url.github.io/ada")
```

Comment:
This needs to be updated with www.ada-url.com

Reply:
you can open a pr to update it.